src/main/kotlin/com/exactpro/th2/read/file/common/AbstractFileReader.kt

```diff
@@ -650,6 +668,8 @@ abstract class AbstractFileReader<T : AutoCloseable>(
             get() = Files.exists(path)
         var truncated: Boolean = false
             private set
+        var readState: ReadState = ReadState.START
```

Comment:
I think we should not expose the state setter (because the FileHolder is not a private class). We can encapsulate the state changes inside some methods